Source/Extensions/Blazorise.RichTextEdit/wwwroot/richtextedit.js

```diff
@@ -1,6 +1,9 @@
 ﻿import "./vendors/quill.js?v=1.4.2.0";
 import { getRequiredElement } from "../Blazorise/utilities.js?v=1.4.2.0";
 
+document.getElementsByTagName("head")[0]
```

Comment:
I don't see this `blazorise.richtextedit.bundle.scp.css` file in the wwwroot. Did you forget to include it?